docs/job/push-to-docker.md

```diff
@@ -18,6 +19,7 @@ workflows:
     jobs:
       - architect/push-to-docker:
           context: "architect"
+          custom_tag: "1.0.0"
```

Comment:
This doesn't sound like a good idea to me. We'll need to change build configuration each time we want to change the value of the tag.

Reply:
yeah I am working in a recipe to install gpu drivers. We had a trouble using sha tags and I would like to do it more deterministic (using driver version as tag value, so customer can set driver version in chat values and it translates easy to container image). But at the same time I want to do it using automation and not building locally. Any sideshow I can do it?

Reply:
I guess this will be something rare but in certain scenarios can be useful

Reply:
Why then not releasing the repo with the gpu dirvers and having nice tag?

Reply:
You can `docker pull quay.io/giantswarm/aws-operator:8.6.0`

Reply:
well I guess it can be an option. I was thinking it is more intuitive if the driver container image is tagged with the driver version it contains (so the customer does need to map versions) like `coreos-nvidia-driver-installer:390.116`